emcc.py

```diff
@@ -974,12 +974,20 @@ def optimizing(opts):
 
     newargs = [a for a in newargs if a is not '']
 
-    # -c means do not link in gcc, and for us, the parallel is to not go all the way to JS, but stop at bitcode
     has_dash_c = '-c' in newargs
+    has_dash_S = '-S' in newargs
     if has_dash_c:
       assert has_source_inputs or has_header_inputs, 'Must have source code or header inputs to use -c'
       target = target_basename + '.o'
```

Comment:
Do we want a `if '-emit-llvm' in newargs` here as well to generate a `.bc` file?